build/test/headless.js

```diff
@@ -0,0 +1,118 @@
+/*global window*/
+const puppeteer = require('puppeteer');
+const getTestPaths = require('./get-test-paths');
+
+(async () => {
+	const testPaths = await getTestPaths();
+
+	if (!testPaths || !testPaths.length) {
+		console.error(`No test paths are specified to run in headless mode.`);
+	}
+
+	/**
+	 * Launch puppeteer
+	 */
+	const browser = await puppeteer.launch({
+		headless: true,
+		args: ['--no-sandbox', '-–disable-setuid-sandbox'],
+		ignoreHTTPSErrors: true,
```

Comment:
Why are we ignoring HTTPS errors?

Reply:
This just skips the warnings generated by using a fixture server that isn't served over TLS. It shouldn't be an issue.

Reply:
Since it stands out, I think we should add a comment explaining _why_ we're doing it.